maat-court-data-api/src/main/java/gov/uk/courtdata/repository/AuthorizationRepository.java

```diff
@@ -0,0 +1,40 @@
+package gov.uk.courtdata.repository;
+
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.Query;
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class AuthorizationRepository {
```

Comment:
Should this be a class & Do we need EntityManager? Can we not achieve this by following the current setup where Repos are interfaces & we run native queries on it?